both/collections/_shared.js

```diff
@@ -135,6 +135,6 @@ getUserAttribute = function(userId, attribute) {
   return getDocumentAttribute(Meteor.users, userId, attribute);
 };
 
-getRoundAttribute = function(roundId, attribute) {
-  return getDocumentAttribute(Rounds, roundId, attribute);
+getRound = function(roundId) {
```

Comment:
Thanks for getting rid of this, what a mess it was.

At this point, it almost seems like we could get rid of this getRound helper method entirely?